docs/contributors/CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing guidelines
```

Comment:
Can we keep this one at the top-level? I think CONTRIBUTING.md at the top-level is used by other systems, simlar to LICENSE, etc.